configs/config.toml.SAMPLE

```diff
@@ -41,7 +41,7 @@
 	# balance below which the client will start reporting errors in logs.
 	# The value should be passed in [wei].
```

Comment:
Let's tweak this file the same way as we did in `keep-core`:
- remove `# The value should be passed in [wei].` comment,
- add examples of how to pass other units.